src/seedu/addressbook/data/person/Address.java

```diff
@@ -45,7 +68,7 @@ public String toString() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Address // instanceof handles nulls
-                && this.value.equals(((Address) other).value)); // state check
+                        && this.value.equals(((Address) other).value)); // state check
```

Comment:
hmmmm, why do you need to still retain `value`?
think what are the potential problems of retaining the value field as it is?